src/Symfony/Component/VarDumper/Caster/MemcachedCaster.php

```diff
@@ -32,12 +35,12 @@ public static function castMemcached(\Memcached $c, array $a, Stub $stub, $isNes
 
     public static function getMemcachedNonDefaultValueOptions(\Memcached $c)
     {
-        $defaultOptions = self::discoverDefaultMemcachedOptions();
-        $optionConstants = self::getMemcachedOptionConstants();
+        self::$defaultOptions = self::$defaultOptions ?? self::discoverDefaultMemcachedOptions();
+        self::$optionConstants = self::$optionConstants ?? self::getMemcachedOptionConstants();
```

Comment:
there is another call to getMemcachedOptionConstants in discoverDefaultMemcachedOptions, looks redundant to me.